### PR TITLE
fix(protocol-designer): remove margins from protocol overview

### PR DESCRIPTION
**Pull request by koji**
# Overview
remove margins from protocol overview by using grid-gap
add paddings to link button (edit*2 + materials list)

### design
https://www.figma.com/design/WbkiUyU8VhtKz0JSuIFA45/Feature%3A-Protocol-Designer-Phase-1?node-id=7269-120559&m=dev

close RQA-3218

## Test Plan and Hands on Testing
- create a new protocol or import a protocol to display protocol overview
check the app's layout aligns with the design

## Risk assessment
low

### CODE REVIEW

**Review by jerader — COMMENTED** [review]
looks like the break point doesn't work anymore. It should go to 1 column once the viewport width is less than 1024px

**Comment by koji** [comment]
@jerader 
break point doesn't work on edge too.

break point did work without Nick's PR?

**Comment by jerader** [comment]
> @jerader break point doesn't work on edge too.
> 
> break point did work without Nick's PR?

@koji oh haha nevermind, just checked, Nick didn't merge his pr in yet https://github.com/Opentrons/opentrons/pull/16239

File: protocol-designer/src/pages/ProtocolOverview/index.tsx [hunk 1]
```diff
@@ -518,28 +510,33 @@ export function ProtocolOverview(): JSX.Element {
...
+            flexDirection={DIRECTION_COLUMN}
+            flex="1"
+            gridGap={SPACING.spacing12}
+          >
             <Flex
-              marginBottom={SPACING.spacing12}
               justifyContent={JUSTIFY_SPACE_BETWEEN}
               alignItems={ALIGN_CENTER}
             >
               <Flex gridGap="30px" alignItems={ALIGN_CENTER}>
```
Comment (jerader) [inline protocol-designer/src/pages/ProtocolOverview/index.tsx]: not introduced in this pr but there is a stray "30px", can you convert to rem?

**Review by jerader — APPROVED** [review]
lgtm!

**Pull request merged**